specification/subscription/resource-manager/Microsoft.Subscription/preview/2018-11-01-preview/subscriptions.json

```diff
@@ -195,7 +128,7 @@
             "description": "The subscription creation parameters.",
             "required": true,
             "schema": {
-              "$ref": "#/definitions/ModernSubscriptionCreationParameters"
```

Comment:
Don't change the existing parameter name, it might be breaking change for existing SDK users